gallery/src/components/NxPagination/NxPaginationPage.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxPaginationExample from './NxPaginationExample';
+import NxPaginationEmptyExample from './NxPaginationEmptyExample';
+import NxPaginationSinglePageExample from './NxPaginationSinglePageExample';
+
+const nxPaginationCode = require('!!raw-loader!./NxPaginationExample').default,
+    nxPaginationEmptyCode = require('!!raw-loader!./NxPaginationEmptyExample').default,
+    nxPaginationSinglePageCode = require('!!raw-loader!./NxPaginationSinglePageExample').default;
+
+const NxPaginationPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Pagination controls are used when there is a great deal of content that needs to be sorted into separate
+        "pages" rather than scrolled. The controls offer multiple methods of navigating the paged content, forward and
+        backward controls, specific page controls, and an ellipsis control.
+        This component renders pagination buttons contained within an <code className="nx-code">.nx-btn-bar</code>.
+        It is intended to be used within an <code className="nx-code">.nx-footer</code> element.
+        There are essentially five different "types" of buttons which can appear within the pagination control:
+      </p>
+      <ul className="nx-list">
+        <li className="nx-list__item">
+          <span className="nx-list__text">Current page button</span>
+          <span className="nx-list__subtext">
+            A button for the current page will always be visible and is distinguished via a dark blue background.
+            Clicking this button has no effect - the component's <code className="nx-code">onChange</code> handler
+            does not get triggered.
+          </span>
+        </li>
+        <li className="nx-list__item">
+          <span className="nx-list__text">Next/previous page buttons</span>
+          <span className="nx-list__subtext">
+            Whenever there are pages either above or below the current page, arrow buttons are rendered as
+            applicable to navigate the pages sequentially.
```

Comment:
```suggestion
            Whenever there are pages either above or below the current page, arrow buttons are rendered and can be used to navigate the pages sequentially.
```

Reply:
>  + Whenever there are pages either above or below the current page, arrow buttons are rendered and can be used to navigate the pages sequentially.

I don't like this suggestion.  I included the phrase "as applicable" deliberately, to imply that you won't get the left arrow when you're on the first page and won't get the right arrow when you're on the last page.